shell/pages/c/_cluster/settings/performance.vue

```diff
@@ -30,6 +30,12 @@ export default {
       this.uiPerfSetting.metadata = { name: SETTING.UI_PERFORMANCE };
     }
 
+    try {
+      this.authUserTTL = await this.$store.dispatch(`management/find`, { type: MANAGEMENT.SETTING, id: SETTING.AUTH_USER_SESSION_TTL_MINUTES });
+    } catch (error) {
+      return;
```

Comment:
This shouldn't return, just swallow the error `} catch { }`, or alternatively log the error.